src/lib/FieldMapper/ContentTranslationFieldMapper/ContentDocumentFulltextFields.php

```diff
@@ -175,7 +175,11 @@ private function doMapFields(Content $content, ContentType $contentType, $langua
      */
     private function doMapRelatedFields(Content $sourceContent, $languageCode, $maxDepth, $depth)
     {
-        $relations = $this->contentHandler->loadRelations($sourceContent->versionInfo->contentInfo->id);
+        $sourceContentId = $sourceContent->versionInfo->contentInfo->id;
```

Comment:
Can we use here getters instead of magic?

Reply:
This is `\Ibexa\Contracts\Core\Persistence\Content` not API Content.